src/eddington_gui/boxes/input_file_box.py

```diff
@@ -115,6 +115,10 @@ def select_file(self, widget):  # pylint: disable=unused-argument
             message=f"Cannot process file with suffix {suffix}",
         )
 
+    @property
+    def selected_sheet(self):
+        return self.__select_sheet
```

Comment:
This is not exactly what I meant. here you return a `toga.Selection` object. I meant that you would return a **string object** which is the name of the selected sheet. It should look something like:
```python
@property
def selected_sheet(self):
    return self.__select_sheet.value

@selected_sheet.setter
def selected_sheet(self, selected_sheet):
    self.__select_sheet.value = selected_sheet
```

After you do that, reuse it everywhere `self.__select_sheet.value` appears. It will make the code much clearer.

Reply:
Actually I tried exactly this, and it didn't behave as expected. 
I'll check why it happens and use a setter